continuous_integration/envs/39-dev.yaml

```diff
@@ -17,8 +17,8 @@ dependencies:
   - hilbertcurve
   # optional dependencies
   - pyarrow
-  - pip
```

Comment:
We should always have `pip` listed among dependencies if we use it to install some. In all envs with `pymorton`. Conda adds it automatically but complains about it.

Reply:
Updated in 07fdecc